src/main/java/ch/uzh/ifi/hase/soprafs24/rest/dto/PlaylistCollectionDTO.java

```diff
@@ -0,0 +1,23 @@
+package ch.uzh.ifi.hase.soprafs24.rest.dto;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
```

Comment:
While @Data is a convenient Lombok annotation it might add a few redundant annotations. As far as i can tell its sufficient just use @Getter and @Setter and @Noargsconstructor